machineconfiguration/v1alpha1/Makefile

```diff
@@ -0,0 +1,4 @@
+.PHONY: test
+test:
+	make -C ../../tests test GINKGO_EXTRA_ARGS=--focus="machineconfiguration.openshift.io/v1alpha1"
+	
```

Comment:
Nit, must be some whitespace on this last line that shouldn't be there, should avoid these red symbols